### PR TITLE
Expose supportsHevcHardwareEncoding

### PR DESCRIPTION
**Pull request by karaggeorge**
Needed for https://github.com/wulkano/kap/issues/341

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -182,3 +182,9 @@ Object.defineProperty(module.exports, 'videoCodecs', {
     return codecs;
   }
 });
+
+Object.defineProperty(module.exports, 'supportsHevcHardwareEncoding', {
+  get() {
+    return supportsHevcHardwareEncoding;
+  }
+});
```
Comment (sindresorhus) [inline index.js]: Why not just `module.exports.supportsHevcHardwareEncoding = supportsHevcHardwareEncoding`?

And I think it should be documented so other can take advantage of it too.

Comment (karaggeorge) [reply index.js]: 😅 thought it was a function call initially, forgot to change it.

Should be fixed and added a readme entry as well

**Comment by sindresorhus** [comment]
Hmm, I just realized we don't need to expose this. From the readme:

> It only returns `hevc` if you're on macOS 10.13 or newer and your computer supports HEVC hardware encoding.
> -\ https://github.com/wulkano/aperture-node#aperturevideocodecs---map

**Comment by karaggeorge** [comment]
Yeah, we can check if the map has that entry. Figured we wanted it more direct. Checking using the map works for me

**Comment by sindresorhus** [comment]
Yeah, I think that's better.

**Pull request closed without merging**